README.md

```diff
@@ -1 +1,58 @@
-# JS-Capstone-Project-
\ No newline at end of file
+# Javascript Capstone Project
+> An API based web app that displays Meals from the MealsDB API and the Involvement API from Microverse. The features of this webapp include
+
+An option to like a meal by clicking on the like button
+A comment button that opens up a modal and allows users to leave comments.
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript(ES6)
+- Webpack
+- Jest Library
+
+## Live Demo
+- at http://localhost:8080/
```

Comment:
- [ ] The live link demo shouldn't be look this, kindly deploy the app to the GitHub page to get the live link demo instead of `localhost:portnumber`

Reply:
Well solved